frontend/src/pages/LandlordPage.tsx

```diff
@@ -69,6 +70,18 @@ const LandlordPage = (): ReactElement => {
     return subscribeLikes(setLikedReviews);
   }, []);
 
+  useEffect(() => {}, [reviewData]);
+
+  type Fields = keyof typeof reviewData[0];
+
+  const sortReviews = (arr: ReviewWithId[], property: Fields) => {
+    let unsorted = arr;
+    return unsorted.sort((r1, r2) => {
+      // @ts-ignore: Object possibly null or undefined
+      return r1?.[property] < r2?.[property] ? 1 : -1;
```

Comment:
I think the reason why it's getting sorted out of order is because `likes` doesn't exist on every `Review` object: check out the picture I attached in the comment, specifically the highlighted part. 
For each `(r1, r2)` being compared, I logged `r1?.id + r2?.id + r1?.[property] + r1?.[property] + (r1?.[property] < r2?.[property] ? 1 : -1)` to see how they are being sorted.  `w5d0lTDUmsai7xkw1kAG` is the id of the review that has `likes: 1`. When compared to a review without property `likes`, e.g. `v42nUJUW9pWFcaI0SQkA` where `r1?.[likes]` evaluates to `undefined`, it gets sorted behind the review with the undefined value when it should be put in front.

Reply:
For the "Sort Reviews By" section, what do you think about making it like this where the text that's displayed is the sorting criteria being used? We could have it default to sort by most recent and either have "Sort By" be static or don't include that.  I'm fine with either option since we don't have a designer right now. 
Option 1: 
<img width="466" alt="Screen Shot 2021-10-30 at 9 53 24 PM" src="https://user-images.githubusercontent.com/57203639/139563458-79ce8be1-6dc1-4828-8bfa-edd7b7d1ad1d.png">
 
Option 2:
![IMG_0393](https://user-images.githubusercontent.com/57203639/139563491-eeb6063a-3ca5-49ec-be5e-022f7a5db0d5.PNG)